ncvoter/urls.py

```diff
@@ -1,7 +1,8 @@
 """ncvoter URL Configuration"""
-from django.conf.urls import url
+from django.conf.urls import url, include
```

Comment:
TIL that the 'new' Django way is to use path, when you don't need regexs and re_path when you do. `url` is an alias to `re_path` and will be deprecated at some point: https://docs.djangoproject.com/en/2.0/ref/urls/#url